src/components/forms/SubmittedFormsList.cy-test.tsx

```diff
@@ -43,7 +43,11 @@ describe("SubmittedFormsList", () => {
     cy.get("[data-cy=noSubmittedFormsMsg]").should("not.exist");
     cy.get(".nhsuk-action-link__text").should(
       "include.text",
-      "form submitted on 22/04/2020"
+      "form submitted on 22/04/2030 00:00" +
+        "form submitted on 22/04/2016 00:00" +
+        "form submitted on 22/04/2012 00:00" +
+        "form submitted on 22/04/2012 00:00" +
+        "form submitted on 22/04/2010 00:00"
```

Comment:
Is it worth having some time variance too? e.g.. `22/04/2012 00:00` `22/04/2012 13:41`